Src/Common/Domain/interface/IVolunteerOpportunity.cs

```diff
@@ -1,18 +1,29 @@
 ﻿using System;
+using System.Collections;
+using System.Collections.Generic;
 
 namespace Ttu.Domain
 {
     public interface IVolunteerOpportunity
     {
 
         // attributes
-        IUser CreatedBy { get; }
+
+        // Who created this project
+        IUser CreatedBy { get; set; }
+        IOrganization Organization { get; set; }
 
         int RecordId { get; }
 
+        string ProjectName { get; set; }

```

Comment:
This could actually remain flattened out on this interface even if you change the VolunteerOpportunity to reference the Project entity. Nice encapsulation if you decide to go that way.